modulemd/foreman_modulemd.yaml

```diff
@@ -0,0 +1,28 @@
+---
+document: modulemd
+version: 2
+data:
+    name: foreman
+    stream: latest
+    version: 0
+    arch: x86_64
+    summary: Foreman module
+    description: >-
+        A module to make Foreman easier to install.
+    license:
+        module:
+            - GPLv3
+    profiles:
+        installer:
+            rpms:
+                - foreman-installer
```

Comment:
What does this mean at runtime? Is this when you run `dnf module install foreman`?

Reply:
`dnf module install foreman/installer` (as it is not the default profile, yet) but yes